src/formatter.js

```diff
@@ -34,13 +37,40 @@ function escapeTeamCityString(str) {
 
 var reportName = 'ESLint Violations';
 
+var options = {
+  details: {
+    default: true
+  },
+  summary: {
+    default: false
+  }
+};
+
+var config = JSON.parse(fs.readFileSync('package.json'))['eslint-teamcity'] || {};
+
+function getOptionValue(optionName) {
+  if (optionName in config) {
+    return config[optionName];
+  } else {
+    return options[optionName].default;
+  }
+}
+
 //------------------------------------------------------------------------------
 // Public Interface
 //------------------------------------------------------------------------------
 module.exports = function(results) {
+
+  var optionDetails = getOptionValue('details');
+  var optionSummary = getOptionValue('summary');
+
   var output = '';
+  var errorCount = 0;
+  var warningCount = 0;
 
-  output += '##teamcity[testSuiteStarted name=\'' + reportName + '\']\n';
+  if (optionDetails) {
```

Comment:
No need to check if optionsDetails exists I think, since it will always be either the user defined value in `package.json` or the default object defined above